test/gst-msdk/util.py

```diff
@@ -88,6 +92,10 @@ def mapformatu(format):
     "AYUV"  : "AYUV",
     "YUY2"  : "YUY2",
     "ARGB"  : "ARGB",
+    "422H"  : "Y42B",
+    "444P"  : "Y444",
```

Comment:
Please just focus on the 422/444 10bit changes in this pull-request.

Reply:
ok, I will revert the commit for 422/444 8bit decoding